wdtk-wikibaseapi/src/main/java/org/wikidata/wdtk/wikibaseapi/WikibaseDataFetcher.java

```diff
@@ -281,6 +283,83 @@ public EntityDocument getEntityDocumentByTitle(String siteKey, String title)
 		return result;
 	}
 
+	/**
+	 * Fetches the MediaInfoId of a page with the given title.
```

Comment:
Only files could have MediaInfo entities. I would make it clearer in the documentation.

An option to make the API simpler is maybe to rename the methods `getMediaInfoIdForFileName` and take for parameter the file name(s) ("Albert Einstein Head.jpg") instead of the file page title ("File:Albert Einstein Head.jpg"). This way, it would not be possible to misuse the API to get a MediaInfo entity for a page that is not a file (e.g. "Main page").

Reply:
Make sense. Seems we only deal with file name at most time, the disability on special pages like "Main Page" may not be a big deal.